tests/components/zha/test_select.py

```diff
@@ -193,7 +193,11 @@ async def test_on_off_select(hass, light, zha_device_joined_restored):
 
     state = hass.states.get(entity_id)
     assert state
-    assert state.state == STATE_UNKNOWN
+    if zha_device_joined_restored.name == "zha_device_joined":
```

Comment:
Generally we try to avoid branching in tests

Reply:
Ok I’ll look tomorrow at using the individual fixtures instead of the combined / parameterized one